config/initializers/rswag-ui.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require 'rswag'
```

Comment:
Idk what's different about how staging and production load dependencies that would need this added statement, but I think this fixes the issue

Reply:
rswag is only in the dev/test groups so I suspect there will be a problem again

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/7295

Reply:
🙏 thanks, reopening in #7298 with that update